lib/chef/chef_fs/data_handler/acl_data_handler.rb

```diff
@@ -11,8 +11,8 @@ def normalize(acl, entry)
             'read' => {},
             'update' => {},
             'delete' => {},
-            'grant' => {}
-            })
+            'grant' => {},
+            },)
```

Comment:
^ there's another edge case here which is less than delightful, but i believe there's another style cop which flags this as a bad way to match up the trailing paren, we just haven't gotten there yet.